index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Awesome Books</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section class="books-container">
+      <div><h1>Awesome Books</h1></div>
+      <div class="books-list">
+        
+      </div>
+    </section>
+    <br/>
+    <form  class="add-books">
+      <input type="text" id="bookName" placeholder="Title" />
+      <br><br>
+      <input type="text" id="bookAuthor" placeholder="Author">
+      <br><br>
```

Comment:
- [ ] [ _OPTIONAL_ ] It's best practice to make form elements required so that an empty form won't be submitted.
- [ ]  [ _OPTIONAL_ ] the `<br>` is a self closing tag just like the `<img/>` tag, Please fix that.